spec.html

```diff
@@ -40009,7 +39979,7 @@ <h1>synchronizes-with</h1>
           </ul>
         </li>
         <li>For each pair (_E_, _D_) in _execution_.[[HostSynchronizesWith]], (_E_, _D_) is in _execution_.[[SynchronizesWith]].</li>
-        <li>For each pair of Synchronize events (_S_, _Sw_) in AgentsSynchronizeWith(_execution_), (_S_, _Sw_) is in _execution_.[[SynchronizesWith]].</li>
+        <li>For each element _eventsRecord_ of _execution_.[[EventsRecords]], for each pair (_S_, _Sw_) in _eventsRecord_.[[AgentSynchronizesWith]], (_S_, _Sw_) is in _execution_.[[SynchronizesWith]].</li>
```

Comment:
Editorial nit: I think it'd be a clearer to have the [[AgentSynchronizesWith]] addition be its own `<li>`

Reply:
done + rebased